lte/gateway/c/core/oai/tasks/sgs/sgs_defs.h

```diff
@@ -28,7 +28,7 @@
 #include <netinet/in.h>
```

Comment:
can you change it
```cpp
pragma once
```

Reply:
Why this header is not used in sgs_task.c?

Reply:
sgs related changes are already done by @VinashakAnkitAman in PR: https://github.com/magma/magma/pull/14339